.github/workflows/deno-build.yml

```diff
@@ -2,8 +2,13 @@ name: deno.land Continuous Deployment
 
 on:
   push:
+    branches:
+      - main
```

Comment:
also..is there a scenario where we'd ever be pushing a release from a branch other than main?

Reply:
I don't think so, but I don't think we want this `branches` section, as this means this workflow will run on any push to `main`. I believe that each entry under `push:` is logically OR'ed together - so with this current workflow file, if _any_ of the following happen, then the deno build would be kicked off:

- a push to main, regardless of tag
- a tag that begins with `@slack/web-api`
  - ... but if a tag matches any of the negation specs (the ones that start with `!`), then the previous match tag would be negated (at least based on the [filtering logic cheatsheet docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet), if I understand them correctly).

It's worth mentioning that a tag is actually its own entity independent of any branch - it's like a reference to a git commit SHA.